instrumentation/runtime/example_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package runtime_test
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"go.opentelemetry.io/contrib/instrumentation/runtime"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/sdk/metric"
+)
+
+func Example() {
+	// This reader is used as a stand-in for a reader that will actually export
+	// data. See https://pkg.go.dev/go.opentelemetry.io/otel/exporters for
+	// exporters that can be used as or with readers.
+	reader := metric.NewManualReader(
+		// Add the runtime producer to get histograms from the Go runtime.
+		metric.WithProducer(runtime.NewProducer()),
```

Comment:
Double-checking (I am not sure). Will it work correctly if the user would like to have multiple metrics  exporters? For instance: OTLP and Prometheus?

Reply:
They would have concurrent access issues, I think.  I can synchronize the goCollector to fix that.

Reply:
fixed